docs/deployment-local-cluster.md

```diff
@@ -0,0 +1,47 @@
+# Deployment on Local Cluster
+
+Kubernetes-nmstate project allows you to spin up a virtualized
+Kubernets/OpenShift cluster thanks to
+[kubevirtci](https://github.com/kubevirt/kubevirtci) project.
+In this guide, we will create a local Kubernetes
+cluster with two nodes and preinstalled node dependencies. Then we will deploy
+kubernetes-nmstate from local sources.
+
+If you want to deploy kubernetes-nmstate on your arbitrary cluster, read
+the [deployment on arbitrary cluster guide](deployment-arbitrary-cluster.md).
+
+Start your local cluster. If you want to use OpenShift instead of Kubernetes or
+a different amount of nodes, check the
+[development guide](developer-guide.md#local-cluster).
+
+```shell
+KUBEVIRT_NUM_NODES=2 make cluster-up
+```
+
+Build kubernetes-nmstate from local sources and install it on the cluster.
+
+```shell
+make cluster-sync
+```
+
+You can ssh into the created nodes using `cluster/cli.sh`.
```

Comment:
cluster-up?

Reply:
Done

Reply:
I don't see the `cluster_up/cli.sh` change